knapsack_rspec_report.json

```diff
@@ -331,7 +329,6 @@
   "spec/features/admin/external_services_spec.rb": 0.3893253803253174,
   "spec/models/stock/package_spec.rb": 0.43047094345092773,
   "spec/controllers/admin/stripe_connect_settings_controller_spec.rb": 0.27138352394104004,
-  "spec/controllers/spree/admin/image_settings_controller_spec.rb": 0.2584362030029297,
```

Comment:
I think this will unbalance our jobs. We might need a full re-balance and update this file.

Reply:
yes, but on another PR. I havent managed to get a new knapsack report the last time I tried, the single job build expired after 3 hours :man_shrugging:

Reply:
sure, I didn't mean to do it here.